src/playbook.yaml

```diff
@@ -25,13 +25,14 @@
         name: docker
         state: started
 
-    - name: Add the ec2-user to the docker group
+    - name: Add the ec2-user and ssm-user to the docker group
       ansible.builtin.user:
         name: "{{ item }}"
         groups: docker
         append: yes
       with_items:
         - ec2-user
+        - ssm-user
```

Comment:
the ssm-user does not exist in an EC2 until someone uses the AWS SSM start-session to login as the `ssm-user`.  I think we would need to pre-emptively create this user or just tell users to always login with the ec2-user which can be done with the [AWS SSM custom commands](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/938836322/Service+Catalog+Provisioning?focusedCommentId=2403598398#SSM-access-with-custom-commands)

Reply:
I prefer converging on `ec2-user`.